src/js/app.js

```diff
@@ -394,7 +391,7 @@ app.controller('walletRecoveryCtrl', function($scope, $q, $modal, $rootScope, $l
             device_name: navigator.userAgent || "Unknown Browser"
         }).then(
             function(result) {
-                var sdk = new blocktrailSDK({apiKey: result.data.api_key, apiSecret: result.data.api_secret, testnet: $scope.recoveryNetwork.testnet});
+                var sdk = new blocktrailSDK({apiKey: result.data.api_key, apiSecret: result.data.api_secret, testnet: $scope.recoveryNetwork.testnet, network: "bch"});
```

Comment:
todo check BCC?

Reply:
It's literally just for the login to retrieve the relevant recovery data - which is network agnostic. 
Basically, the network is of no importance in this step.

I've still added some differentiation given only the two networks which are supported.